packages/cli/templates/defaults/app/constants.ts

```diff
@@ -1,2 +1,24 @@
+import type { ImageLoader } from "@webstudio-is/image";
+
 export const assetBaseUrl = "/assets/";
 export const imageBaseUrl = "/assets/";
+
+export const imageLoader: ImageLoader = ({ quality, src, width }) => {
+  if (process.env.NODE_ENV !== "production") {
+    return imageBaseUrl + src;
+  }
+
+  const searchParams = new URLSearchParams();
+  searchParams.set("w", width.toString());
+  searchParams.set("q", quality.toString());
```

Comment:
Looks like the `searchParams` is not being used 🤔

Reply:
Thanks

Reply:
Strange that checks completed

Reply:
Maybe they are considering the `searchParams.set` as declared and used !!